content/en/user-manual/assets/import-pipeline/index.md

```diff
@@ -33,18 +27,30 @@ When you update an source asset by uploading a new version of the file. There ar
 
 So, if you leave this enabled, you are able to organize your source and target assets into folders and be sure that when you update a source assets it will update all related assets.
 
-## Texture Options
+### Assets default to preload
+
+Newly created assets will automatically be set to [preload][2] or not depending on whether this option is enabled or not. The exception to this are Javascript script files which will always be set to preloaded when created.
 
-These options only affect the importing of images and textures
+## Texture Import Settings
+
+These options only affect the importing of images and textures.
 
 ### Texture POT (Power of Two)
 
 When this option is enabled textures that are not a power of two will be converted to the nearest power of two resolution when they are imported.
 
-## Model Options
+### Create Atlases
+
+Images that are uploaded will be imported as a texture atlas instead of a normal texture asset. This is useful time saver when uploading many spritesheets or UI assets.
```

Comment:
```suggestion
Images that are uploaded will be imported as a texture atlas instead of a normal texture asset. This is a useful time saver when uploading many spritesheets or UI assets.
```